song-server/src/main/java/bio/overture/song/server/security/StudySecurity.java

```diff
@@ -70,15 +72,28 @@ public boolean authorize(Authentication authentication, String studyId) {
 
     Set<String> grantedScopes;
 
-    if ("keycloak".equalsIgnoreCase(provider)) {
+    if (authentication instanceof BearerTokenAuthentication) {
```

Comment:
Let's be explicit that this is meant to be done for pcglauthz only, be consistent and use the same pattern as with the keycloak provider.
```suggestion
    if ("pcglauthz".equalsIgnoreCase(provider) && authentication instanceof BearerTokenAuthentication) {
```